app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/components/InstructionRowItem.kt

```diff
@@ -0,0 +1,120 @@
+package com.x8bit.bitwarden.ui.vault.feature.importlogins.components
+
+import android.content.res.Configuration
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.x8bit.bitwarden.R
+import com.x8bit.bitwarden.ui.platform.components.card.BitwardenContentCard
+import com.x8bit.bitwarden.ui.platform.components.util.rememberVectorPainter
+import com.x8bit.bitwarden.ui.platform.theme.BitwardenTheme
+import com.x8bit.bitwarden.ui.vault.feature.importlogins.model.InstructionStep
+
+/**
+ * Row item for the content card of the import logins screen instructions.
+ */
+@Composable
+fun InstructionRowItem(
```

Comment:
I see you made `BitwardenContentCard` generic, but this should be too.

It is this [component](https://www.figma.com/design/gAE9D9OP2zOuVU7aSiIbHy/Bitwarden-Android-Mobile-App-Reskin---V3-(current)?node-id=70193-261651&t=IlzZk0guvtAHkHuY-4).